### ISSUE TITLE
broken link

### ISSUE THREAD

**Issue by alisade** [weaveworks/eksctl#1596]
https://eksctl.io/usage/eks-managed-nodegroups/eks-user-guide

**Comment by alisade** [comment]
```bash
curl https://eksctl.io/usage/eks-managed-nodegroups/eks-user-guide
<!doctype html><html lang=en><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><title>404 Page not found
| eksctl</title><link rel=stylesheet href=/book.min.e5bc17c1b0de692baa500135e5ae259468213bf7a6c012b1416f659a2974f5ab.css><link rel=icon href=/assets/favicon/favicon_150px.png type=image/x-icon><link rel=stylesheet href=/assets/styles/github-gist.css><script src=/assets/highlight.pack.js></script><script>hljs.initHighlightingOnLoad();</script></head><body><main class="flex justify-center"><h1>404 Not Found</h1></main></body></html>```

**Comment by mumoshu** [comment]
This is resolved via #1598. Thanks for reporting @alisade!
